### PR TITLE
Clean up router class documentation.

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: app/models/requests/router.rb [hunk 1]
```diff
@@ -12,23 +13,25 @@ def initialize(requestable:, user:, any_loanable: false)
...
-    # :paging
-    # :trace
-
-    # user levels
-    # guest - Access patron - shouldn't show recap_edd
-    # barcode - no ill, no bd
-    # cas - all services
+    # Current Service Types Assigned
+    # :online - material is available online at a URL
+    # :aeon - mataerial is stored in a location where it can be requested via Aeon
```
Comment (christinach) [inline app/models/requests/router.rb:18]: material

File: app/models/requests/router.rb [hunk 2]
```diff
@@ -12,23 +13,25 @@ def initialize(requestable:, user:, any_loanable: false)
...
+    # :annex - material is stored in an Annex location
+    # :on_shelf - material is stored in a campus library location
+    # :on_shelf_edd - material is in a campus library location that is eligible for digitization
+    # :on_order - material has a status in Alma that indicates it is ordered but has not yet arrived on campus
+    # :in_process - material has a status in Alma that indicates it has arrived on campus but has not been processed and shelved
+    # :recap - material is stored at recap; can be paged to campus and circulates
+    # :recap_in_library - material is stored at recap; can be paged to campus, but does not circulate
+    # :recap_edd - material is stored in a recap location that permits digitization
+    # :recap_no_items - material in a recap location with item record data
+    # :ill - material has a status in Alma making it unavailable for ciruclation and is in a location that is eligible for resource sharing
```
Comment (christinach) [inline app/models/requests/router.rb:28]: circulation

**Review by christinach — COMMENTED** [review]
@kevinreiss thanks! two typos.

**Comment by kevinreiss** [comment]
Related to #3432

File: app/models/requests/router.rb [hunk 3]
```diff
@@ -12,23 +13,25 @@ def initialize(requestable:, user:, any_loanable: false)
...
+    # :annex - material is stored in an Annex location
+    # :on_shelf - material is stored in a campus library location
+    # :on_shelf_edd - material is in a campus library location that is eligible for digitization
+    # :on_order - material has a status in Alma that indicates it is ordered but has not yet arrived on campus
+    # :in_process - material has a status in Alma that indicates it has arrived on campus but has not been processed and shelved
+    # :recap - material is stored at recap; can be paged to campus and circulates
+    # :recap_in_library - material is stored at recap; can be paged to campus, but does not circulate
+    # :recap_edd - material is stored in a recap location that permits digitization
+    # :recap_no_items - material in a recap location with item record data
+    # :ill - material has a status in Alma making it unavailable for circlation and is in a location that is eligible for resource sharing
```
Comment (christinach) [inline app/models/requests/router.rb:28]: circulation

Comment (christinach) [reply app/models/requests/router.rb:28]: Thanks @kevinreiss . One typo.

Comment (maxkadel) [inline app/models/requests/router.rb:28]:
```suggestion
    # :ill - material has a status in Alma making it unavailable for circulation and is in a location that is eligible for resource sharing
```

**Review by christinach — APPROVED** [review]
Thanks @kevinreiss

**Pull request merged**